core/themes/seven/css/style.css

```
@@ -1032,7 +1032,7 @@ button.button-primary:focus {
 }
 a.button-secondary,
 input.button-secondary,
-button.button-secondary {
+button.button-secondary{
```

```suggestion
button.button-secondary {
```

Adding this as a code suggestion. A core committer should be able to add it as a commit before merging this PR.